dkms build failure on linux kernel 6.8.0-49.generic

It seems that topology_num_cores_per_package is supported from kernel version 6.9, but not in 6.8.

/var/lib/dkms/amd_energy/1.0/build/amd_energy.c:243:26: error: implicit declaration of function ‘topology_num_cores_per_package’; did you mean ‘topology_max_die_per_package’? [-Werror=implicit-function-declaration]
  243 |         sockets = cpus / topology_num_cores_per_package();
      |                          ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
      |                          topology_max_die_per_package

--- a/amd_energy.c
+++ b/amd_energy.c
@@ -236,7 +236,7 @@ static int amd_create_sensor(struct device *dev,
 	 * c->x86_max_cores is the linux count of physical cores
 	 * total physical cores/ core per socket gives total number of sockets.
 	 */
-#if LINUX_VERSION_CODE <= KERNEL_VERSION(6, 8, 0)
+#if LINUX_VERSION_CODE <= KERNEL_VERSION(6, 9, 0)
 	struct cpuinfo_x86 *c = &boot_cpu_data;
 	sockets = cpus / c->x86_max_cores;
 #else

